ansible_collections/arista/avd/roles/eos_designs/templates/base/local-users.j2

```diff
@@ -4,10 +4,13 @@ local_users:
     privilege: {{ local_users[local_user].privilege }}
 {%     if local_users[local_user].role is defined %}
     role: {{ local_users[local_user].role }}
-{%      endif %}
-{%    if local_users[local_user].sha512_password is defined %}
+{%     endif %}
+{%     if local_users[local_user].sha512_password is defined %}
```

Comment:
Please use `arista.avd.defined`. Also please update the rest of this file. Thanks.